pyproject.toml

```diff
@@ -17,31 +20,31 @@ python = ">=3.9,<3.14"
 click = "*"
 platformdirs = "*"
 PySide6 = "^6.7.1"
-PyMuPDF = "^1.23.3"  # The version in Fedora 39
+PyMuPDF = "^1.23.3"                           # The version in Fedora 39
 colorama = "*"
-pyxdg = {version = "*", platform = "linux"}
+pyxdg = { version = "*", platform = "linux" }
 requests = "*"
 markdown = "*"
 packaging = "*"
 
 [tool.poetry.scripts]
 dangerzone = 'dangerzone:main'
 dangerzone-cli = 'dangerzone:main'
-#assets = 'assets:main'
+dangerzone-image = "dangerzone.updater.cli:main"
 
 # Dependencies required for packaging the code on various platforms.
 [tool.poetry.group.package.dependencies]
 setuptools = "*"
-cx_freeze = {version = "^7.2.5", platform = "win32"}
-pywin32 = {version = "*", platform = "win32"}
-pyinstaller = {version = "*", platform = "darwin"}
+cx_freeze = { version = "^7.2.5", platform = "win32" }
+pywin32 = { version = "*", platform = "win32" }
+pyinstaller = { version = "*", platform = "darwin" }
 doit = "^0.36.0"
 jinja2-cli = "^0.8.2"
-assets = { git = "https://github.com/freedomofpress/assets", tag = "0.1.0" }
+mazette = { git = "https://github.com/freedomofpress/mazette", tag = "0.2.0" }
```

Comment:
```suggestion
mazette = "^0.2.0"
```

Reply:
Well, I'm not sure that would work, since the currently tagged version (0.2.0) is not pushed on PyPI.

But... That could be better to actually release something out and then refer to it rather than pointing to the tags manually.

I remember a discussion in Bxl where you mentioned that you would like us to avoid having to go the full route of releasing to PyPI to be able to use that in our project.

Let me know what you prefer and I can publish it if needed.